scripts/convert_diffusers_sdxl_lora_to_webui.py

```diff
@@ -0,0 +1,79 @@
+# Script for converting a HF Diffusers trained SDXL LoRAs (be it in the old, new or PEFT format)
```

Comment:
I'd add some reference examples here too as I believe this is going to be used quite a lot by the community ❤️

Reply:
Done!